frontend/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -36,6 +36,224 @@ const connector = connect(
   }
 );
 
+// TODO: this button is placed at the bottom of the menu,
+// But I am able to scroll down.. so it's not at the bottom.
+// If someone could help with the height stuff so there's no scrolling that would be great
+const SignOutButton = styled.button`
```

Comment:
I'd recommend using flex here. If you do a `space-between` for the `justify-content` it will put it at the bottom of the view.

Reply:
Hmm I did that, but it placed the log out button right underneath the red menu box div. (I also removed the position: absolute; bottom 0px; from the sign out button)